### ISSUE TITLE
jar defaults

### ISSUE THREAD

**Issue by jakwuh** [popomore/koa-proxy#44]
Hi,

TLDR;
Maybe `jar = false` by default is a better decision?

---

It seems to me `jar = true` is not such a good default value.
Last time I spent plenty of hours while debugging our app, using wireshark / tcpdump on servers and making binary search on code because I was ran out of any ideas.

`koa-proxy` is for node.js. And usually `set-cookie` header contains user-grained cookies, such as `csrftoken`, `user_id` and so on. By setting `jar = true` by default you implicitly save cookies **for** user A and then send them together with cookies **from** user B. This is weird to me.

I will be very happy if I would be the latest who completed such a quest understanding what is happening.
